MSAL/src/native_auth/network/responses/validator/sign_in/validated_response/MSALNativeAuthSignInChallengeValidatedResponse.swift

```diff
@@ -59,7 +59,7 @@ enum MSALNativeAuthSignInChallengeValidatedErrorType: Error {
         }
     }
 
-    func convertToSignInPasswordStartError() -> SignInPasswordStartError {
+    func convertToSignInPasswordStartError() -> SignInStartError {
```

Comment:
This method name seems a little bit unclear to me now, we're saying that this method converts to SignInPassword but actually it converts it to SignInStartError. It's seems a little bit unconsistent to me

Reply:
These two methods were almost identical, removed one